qiita_pet/handlers/user_handlers.py

```diff
@@ -93,8 +93,11 @@ def post(self):
             info = user.info
             try:
                 send_email(user.id, "Qiita: Password Reset", "Please go to "
-                           "the following URL to reset your password: "
-                           "http://qiita.colorado.edu/auth/reset/%s" %
+                           "the following URL to reset your password: \n"
+                           "http://qiita.colorado.edu/auth/reset/%s  \nYou "
```

Comment:
The hostname here should be sourced from the configuration file as we cannot assume qiita is running from under colorado.edu